weld/lib.rs

```diff
@@ -312,14 +285,29 @@ pub unsafe extern "C" fn weld_module_run(module: *mut WeldModule,
 }
 
 #[no_mangle]
+/// Returns memory usage of a Weld module. This includes unfreed memory allocated over
+/// all prior runs of the module.
+pub unsafe extern "C" fn weld_module_memory_usage(module: *mut WeldModule) -> libc::int64_t {
+    let module = &mut *module;
+    module.run_named("rt_memory_usage", 0).unwrap()
+}
+
+#[no_mangle]
+/// Frees all unfreed memory allocated by this module over all prior runs.
+pub unsafe extern "C" fn weld_module_mem_free(module: *mut WeldModule) {
+    let module = &mut *module;
+    module.run_named("rt_run_free", 0).unwrap();
+}
```

Comment:
So is the idea that you call weld_module_mem_free to free stuff it allocated but leave the module around, and weld_module_free to delete the whole module? It seems a bit weird to tie memory management to a module. For example, can you call the same module from multiple threads now?